File: NetBeans_IDE/JsonPL/src/com/middlemind/JsonPL/ProcessCallRunner.java
package com.middlemind.JsonPL;

import com.middlemind.JsonPL.JsonObjs.JsonObjSysBase;

/**
 *
 * @author brusc
 */
public class ProcessCallRunner implements Runnable {
    private final JsonObjSysBase lcall;
    private final JsonObjSysBase func;
    private final JsonPlState main;
    
    public ProcessCallRunner(JsonObjSysBase Lcall, JsonObjSysBase Func, JsonPlState Main) {
        lcall = Lcall;
        func = Func;
        main = Main;
    }
    
    @Override
    public void run() {
        main.processCall(lcall, func);
    }    
}

File: storage/alt_interpreters/VisualStudio_IDE/JsonPL/JsonPL/com/middlemind/JsonPL/ProcessCallRunner.cs
using com.middlemind.JsonPL.JsonObjs;
using System;
using System.Collections.Generic;
using System.Text;

namespace com.middlemind.JsonPL
{
    public class ProcessCallRunner
    {
        private JsonObjSysBase lcall;
        private JsonObjSysBase func;
        private JsonPlState main;

        public ProcessCallRunner(JsonObjSysBase Lcall, JsonObjSysBase Func, JsonPlState Main)
        {
            lcall = Lcall;
            func = Func;
            main = Main;
        }

        public void run()
        {
            main.processCall(lcall, func);
        }
    }
}
